packaging/os/slackpkg.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2014, Kim Nørgaard
+# Written by Kim Nørgaard <jasen@jasen.dk>
+# Based on pkgng module written by bleader <bleader@ratonland.org>
+# that was based on pkgin module written by Shaun Zinck <shaun.zinck at gmail.com>
+# that was based on pacman module written by Afterburn <http://github.com/afterburn> 
+#  that was based on apt module written by Matthew Williams <matthew@flowroute.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+
+DOCUMENTATION = '''
+---
+module: slackpkg
+short_description: Package manager for Slackware >= 12.2
+description:
+    - Manage binary packages for Slackware using 'slackpkg' which
+      is available in versions after 12.2.
+version_added: "1.6"
+options:
+    name:
+        description:
+            - name of package to install/remove
+        required: true
+
+    state:
+        description:
+            - state of the package
+        choices: [ 'present', 'absent', 'installed', 'removed', 'latest' ]
+        required: false
+        default: present
+
+    update_cache:
+        description:
+            - update the package database first
+        required: false
+        default: false
+        choices: [ true, false ]
+
+author: Kim Nørgaard
```

Comment:
author format changed, please add your GitHub handle to the author e.g `author:  Kim Nørgaard (@KimNorgaard)"`